diff --git a/module/network/HTTPRequest.py b/module/network/HTTPRequest.py
index d6fcae7..b841481 100644
--- a/module/network/HTTPRequest.py
+++ b/module/network/HTTPRequest.py
@@ -78,6 +78,7 @@ class HTTPRequest():
         self.c.setopt(pycurl.NOPROGRESS, 1)
         if hasattr(pycurl, "AUTOREFERER"):
             self.c.setopt(pycurl.AUTOREFERER, 1)
+        self.c.setopt(pycurl.SSL_CIPHER_LIST, "RC4-MD5:DEFAULT")
         self.c.setopt(pycurl.SSL_VERIFYPEER, 0)
         self.c.setopt(pycurl.LOW_SPEED_TIME, 60)
         self.c.setopt(pycurl.LOW_SPEED_LIMIT, 5)
